### ISSUE TITLE
Integration with Google Calendar

### ISSUE THREAD

**Issue by granalberto** [opencats/OpenCATS#442]
It would be great if you can integrate with Google Calendar.

**Comment by RussH** [comment]
You can export the OpenCATS iCal feed to view on Google calender.

**Comment by granalberto** [comment]
> You can export the OpenCATS iCal feed to view on Google calender.

Thanks for the tip. Still would be great to have this instegration, don't you think?

**Comment by RussH** [comment]
Yes, but unless someone provides it, there are allot of features that'd be great..!

**Comment by shantadam** [comment]
Alberto,

In fact, work is in-progress to have the integration module.

Regards,
Shant Adam

**Comment by RussH** [comment]
@granalberto Google Calendar integration would be possible, but it isn't currently on the OpenCATS roadmap and isn't something we're likely to deliver in the near future. Closing this for now rather than leaving the feature request open indefinitely.
